apps/opencs/view/render/worldspacewidget.hpp

```diff
@@ -35,6 +36,14 @@ namespace CSVRender
     class CellArrow;
     class EditMode;
 
+    struct WorldspaceHitResult
+    {
+        bool hit;
+        osg::ref_ptr<TagBase> tag;
+        unsigned int i0, i1, i2;
+        osg::Vec3d worldPos;
+    };
```

Comment:
Why do we need this struct? The tag based approach seems to work fine. Also, i0, i1 and i2 are a bit cryptic as variable names go.